docs/InspectingContracts.md

```diff
@@ -0,0 +1,73 @@
+
+```json
+{
+    "owed_payment": {
+        "transmitter": "TRANSMITTER_ADDRESS"
+    }
+}
+```
+
+Deployed contracts for Terra can be found in the [Reference Data Directory](https://github.com/smartcontractkit/reference-data-directory). This directory includes deployments of proxies and OCR2 contracts for both [mainnet](https://github.com/smartcontractkit/reference-data-directory/blob/master/directory-terra-mainnet.json) and [testnet](https://github.com/smartcontractkit/reference-data-directory/blob/master/directory-terra-testnet-bombay.json).
+
+### Making Queries Via Terra Finder
+
+On the [Terra Finder](https://finder.terra.money/) home page, search the address of the contract that you want to inspect. For example, to inspect the OCR2 contract for the USD/USDT feed on testnet, search *terra14mf0qcjpduhcs8p6289pjnwn8skhgk5aus3yxg*. Make sure you switch to testnet in the top right hand corner of the home page before you search.
+
+Once on the smart contract page, you can create a JSON query. Below is a valid JSON query to the OCR2 contract, where *terra1myd0kxk3fqaz9zl47gm2uvxjm0zn3lczrtvljz* is an address that has performed a transaction with the contract. Transactions can also be viewed in the smart contract page.
+
+```json
+{
+    "owed_payment": {
+        "transmitter": "terra1myd0kxk3fqaz9zl47gm2uvxjm0zn3lczrtvljz"
+    }
+}
+```
+
+Note that you can only make JSON queries in [Terra Finder](https://finder.terra.money/). If the query is a string, use cURL.
+
+### Making Queries Via cURL
+
+You can also interact with deployed contracts via cURL. Some examples of querying the USD/USDT proxy contract can be found below.
+
+```bash
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="owner"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="decimals"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="version"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="description"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="latest_round_data"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="proposed_latest_round_data"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="aggregator"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="phase_id"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg="proposed_aggregator"'
+curl 'https://bombay-fcd.terra.dev/wasm/contracts/terra134lgzqfwms0sg4a33wpygj8waff2d704gcezsu/store?query_msg=%7B%22round_data%22:%7B%22round_id%22:4294968601%7D%7D'
```

Comment:
Explain that this is an URL encoded JSON: `{"round_data":{"round_id":4294968601}}`